workflows/deleting-a-workflow.md

```diff
@@ -12,7 +12,9 @@ On the Aqueduct UI, navigate to the settings pane by clicking on the gear icon o
 
 ![](<../.gitbook/assets/image (6) (1).png>)
 
-After workflow deletion has succeeded, you will be redirected to the workflows page.
+You can delete the workflow by following the instructions. You are given the option to delete the entirety of any objects saved by the workflow. 
```

Comment:
I think we could add a bit more detailed here. The word "entirety" is carrying a lot of weight, which we should explain further. I would say two separate things: (1) you can choose which objects to delete; (2) any objects you pick will be completely deleted.